### PR TITLE
Added fireservicerota documentation 

### PR DESCRIPTION
**Pull request by cyberjunky**
## Proposed change

This PR adds documentation for the new fireservicerota integration.

I was forced to close the old PR https://github.com/home-assistant/home-assistant.io/pull/14568  because I want to contribute to other issues, and open PR's for current branch again, and I used 'current' as fork branch preventing this.
Sorry for the inconvenience.

It now also points to 'next' branch instead of 'current'.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/38206
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by cyberjunky** [comment]
Created a mess, I will try to clean up ones I know how ❓

File: source/_integrations/fireservicerota.markdown [hunk 1]
```diff
@@ -0,0 +1,220 @@
+---
+title: FireServiceRota
+description: Instructions on how to configure the FireServiceRota integration for Home Assistant.
+ha_category:
+  - Binary Sensor
+  - Sensor
+  - Switch
+ha_iot_class: Cloud Polling
+ha_release: 0.118
```
Comment (balloob) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
ha_release: 0.119
```

**Review by balloob — DISMISSED** [review]

File: source/_integrations/fireservicerota.markdown [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+ha_codeowners:
+  - '@cyberjunky'
+ha_config_flow: true
+ha_domain: fireservicerota
+---
+
+FireServiceRota is a powerful and flexible availability, scheduling and dispatching system for firefighters.
+It's the international brand of the Dutch [BrandweerRooster](https://www.brandweerrooster.nl), which is in use by more than 200 fire stations in The Netherlands.
+
+The `fireservicerota` integration provides you realtime information about incidents (emergency calls) from your local fire station, and the ability to send a response depending on your duty schedule.
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
The FireServiceRota integration provides you real-time information about incidents (emergency calls) from your local fire station and the ability to send a response depending on your duty schedule.
```

File: source/_integrations/fireservicerota.markdown [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+A word of caution: Do not solely rely on this integration for your emergency calls!
+
+</div>
+
+This integration provides the following platforms:
+
+* Sensor: Incoming emergency calls. Metadata contains _among other data_ the location of the incident and a text-to-speech URL. The integration uses a WebSocket client connection with the service to ensure a minimum delay.
+* Binary Sensor: Your current duty status (as scheduled via the FireServiceRota mobile app and/or website)
+* Switch: Enabled for 30 minutes after any emergency call. ‘on’ represents a confirmed response. Use this to automate your emergency call response and save valuable seconds.
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
- Sensor: Incoming emergency calls. Metadata contains _among other data_ the location of the incident and a text-to-speech URL. The integration uses a WebSocket client connection with the service to ensure a minimum delay.
- Binary Sensor: Your current duty status (as scheduled via the FireServiceRota mobile app and/or website).
- Switch: Enabled for 30 minutes after an emergency call. ‘on’ represents a confirmed response. Use this to automate your emergency call response and save valuable seconds.
```

File: source/_integrations/fireservicerota.markdown [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+<img src='/images/integrations/fireservicerota/dashboard.png' alt='Example of a FireServiceRota dashboard' class="no-shadow"/> 
+
+This screenshot shows what a FireServiceRota dashboard can look like.
+
+
+## Debugging
+
+The FireServiceRota integration will log additional information about websocket incidents received, response and duty status gathered, and other messages when the log level is set to `debug`. Add the the relevant lines below to the `configuration.yaml`:
+
+```yaml
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown:218]: This code block is incomplete / not closed?

File: source/_integrations/fireservicerota.markdown [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+### Screenshot
+
+<img src='/images/integrations/fireservicerota/dashboard.png' alt='Example of a FireServiceRota dashboard' class="no-shadow"/> 
+
+This screenshot shows what a FireServiceRota dashboard can look like.
+
+
+## Debugging
+
+The FireServiceRota integration will log additional information about websocket incidents received, response and duty status gathered, and other messages when the log level is set to `debug`. Add the the relevant lines below to the `configuration.yaml`:
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
The FireServiceRota integration will log additional information about WebSocket incidents received, response and duty status gathered, and other messages when the log level is set to `debug`. Add the relevant lines below to the `configuration.yaml`:
```

File: source/_integrations/fireservicerota.markdown [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+```
+
+{% endraw %}
+
+### Screenshot
+
+<img src='/images/integrations/fireservicerota/dashboard.png' alt='Example of a FireServiceRota dashboard' class="no-shadow"/> 
+
+This screenshot shows what a FireServiceRota dashboard can look like.
+
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown:213]:
```suggestion
```

File: source/_integrations/fireservicerota.markdown [hunk 7]
```diff
@@ -0,0 +1,220 @@
...
+1. From Home Assistant, navigate to ‘Configuration’ then ‘Integrations’. Click the plus icon and type/select ‘FireServiceRota’.
+1. Enter your login credentials.
+1. Choose your platform `www.brandweerrooster.nl` or `www.fireservicerota.co.uk`.
+1. Click the Save button.
+
+## Entities
+
+The following entities are created:
+
+### sensor.incidents
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: These entity ID's don't have to match (e.g., renamed, or changed by the end-user). Therefore a descriptive name might be more fitting.

File: source/_integrations/fireservicerota.markdown [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+
+| Value | Description |
+| ----- | ----------- |
+| `value` | Incident text message.|
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: This is like a bit of a weird table. Maybe it makes more sense to describe the sensor value in normal text in this case.

File: source/_integrations/fireservicerota.markdown [hunk 9]
```diff
@@ -0,0 +1,220 @@
...
+| `created_at` | Date and time when incident was created.|
+| `message_to_speech_url` | The URL of the mp3 file containing the spoken text of the incident.|
+| `prio` | Priority of the incident, 'a1', 'a2', 'b1' or 'b2'.|
+| `type` | Type of incident, e.g. 'incident_alert'.|
+| `responder_mode` | Modes of response, e.g. 'available_in_schedule_is_acknowledgment'.|
+| `can_respond_until` | Date and time until response is accepted.|
+| `latitude` | Latitude.|
+| `longitude` | Longitude.|
+| `address_type` | Type of address, e.g. 'home'.|
+| `formatted_address` | Address in string format.|
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
| Attribute | Description |
| --------- | ----------- |
| `trigger` | Type of trigger, `new` or `update`.|
| `state` | The state of the incident. |
| `created_at` | Date and time when incident was created.|
| `message_to_speech_url` | The URL of the mp3 file containing the spoken text of the incident.|
| `prio` | Priority of the incident, `a1`, `a2`, `b1` or `b2`.|
| `type` | Type of incident, e.g., `incident_alert`.|
| `responder_mode` | Modes of response, e.g., `available_in_schedule_is_acknowledgment`.|
| `can_respond_until` | Date and time until response is accepted.|
| `latitude` | Latitude.|
| `longitude` | Longitude.|
| `address_type` | Type of address, e.g., `home`.|
| `formatted_address` | Address in string format.|

```

File: source/_integrations/fireservicerota.markdown [hunk 10]
```diff
@@ -0,0 +1,220 @@
...
+### binary_sensor.duty
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: Same as above; this doesn't have to match, recommended to use a descriptive name.

File: source/_integrations/fireservicerota.markdown [hunk 11]
```diff
@@ -0,0 +1,220 @@
...
+
+| Value | Description |
+| ----- | ----------- |
+| `value` | 'On' = on duty. 'Off' = no duty scheduled.|
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: Same as above.

File: source/_integrations/fireservicerota.markdown [hunk 12]
```diff
@@ -0,0 +1,220 @@
...
+| Attribute | Description |
+| --------- | ----------- |
+| `start_time` | Start date and time of duty schedule.|
+| `end_time` | End date and time of duty schedule.|
+| `available` | 'true of 'false'.|
+| `active` | 'true of 'false'.|
+| `assigned_function_ids` | Function id's, e.g. '540'.|
+| `skill_ids` | Skill id's, e.g. '6, 8'.|
+| `type` | Type, e.g. 'standby_duty'.|
+| `assigned function` | Assigned function, e.g. 'Chauffeur'.|
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
| Attribute | Description |
| --------- | ----------- |
| `start_time` | Start date and time of duty schedule.|
| `end_time` | End date and time of duty schedule.|
| `available` | `true` or `false`.|
| `active` | `true` or `false`.|
| `assigned_function_ids` | Function ID's, e.g., `540`.|
| `skill_ids` | Skill ID's, e.g., `6, 8`.|
| `type` | Type, e.g., `standby_duty`.|
| `assigned function` | Assigned function, e.g., `Chauffeur`.|
```

File: source/_integrations/fireservicerota.markdown [hunk 13]
```diff
@@ -0,0 +1,220 @@
...
+
+### switch.incident_response
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: Same as above

File: source/_integrations/fireservicerota.markdown [hunk 14]
```diff
@@ -0,0 +1,220 @@
...
+
+| Value | Description |
+| ----- | ----------- |
+| `value` | 'On' = response acknowledged. 'Off' = response rejected.|
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: Same as the above value sections

File: source/_integrations/fireservicerota.markdown [hunk 15]
```diff
@@ -0,0 +1,220 @@
...
+| `user_name` | Your username.|
+| `assigned_skill_ids` | Assigned skill id's.|
+| `responded_at` | Time responded.|
+| `start_time` | Incident repsonse start time.|
+| `status` | Status of repsonse, e.g. 'pending'.|
+| `reported_status` | Reported status, e.g. 'shown_up'.|
+| `arrived_at_station` | 'true' or 'false'.|
+| `available_at_incident_creation` | 'true' or 'false'.|
+| `active_duty_function_ids` | Active function id's, e.g. '540'.|
+
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
| Attribute | Description |
| --------- | ----------- |
| `user_name` | Your username.|
| `assigned_skill_ids` | Assigned skill ID's.|
| `responded_at` | Time responded.|
| `start_time` | Incident repsonse start time.|
| `status` | Status of repsonse, e.g., `pending`.|
| `reported_status` | Reported status, e.g., `shown_up`.|
| `arrived_at_station` | `true` or `false`.|
| `available_at_incident_creation` | `true` or `false`.|
| `active_duty_function_ids` | Active function ID's, e.g., `540`.|
```

File: source/_integrations/fireservicerota.markdown [hunk 16]
```diff
@@ -0,0 +1,220 @@
...
+
+## Advanced Configuration
+
+With Automation you can configure one or more of the following useful actions:
+
+1. Sound an alarm and/or switch on lights when an emergency incident is received.
+1. Use text to speech to play incident details via a mediaplayer while getting dressed.
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
1. Use text to speech to play incident details via a media player while getting dressed.
```

File: source/_integrations/fireservicerota.markdown [hunk 17]
```diff
@@ -0,0 +1,220 @@
...
+1. Respond with an response acknowledgement using a door-sensor when leaving the house or by pressing a button to let your teammates know you are underway.
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]:
```suggestion
1. Respond with a response acknowledgment using a door-sensor when leaving the house or by pressing a button to let your teammates know you are underway.
```

File: source/_integrations/fireservicerota.markdown [hunk 18]
```diff
@@ -0,0 +1,220 @@
...
+    action:
+      service: light.turn_on
+      entity_id: light.bedroom
+
+  - alias: 'Play TTS incident details when incident is received'
+    trigger:
+      platform: state
+      entity_id: sensor.incidents
+      attribute: message_to_speech_url
+    condition: "{{ state_attr('sensor.incidents', 'message_to_speech_url') != 'None' }}"
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: Conditions can work with attributes too!

```yaml
condition: not
conditions:
  condition: state
  entity_id: sensor.incidents
  attribute: message_to_speech_url
  state: "None"
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thanks for the PR @cyberjunky, I've left some review comments. Could you please take a look? Thanks 👍

File: source/_integrations/fireservicerota.markdown (hunk 4 above)
Comment (cyberjunky) [reply source/_integrations/fireservicerota.markdown:218]: Fixed

File: source/_integrations/fireservicerota.markdown (hunk 7 above)
Comment (cyberjunky) [reply source/_integrations/fireservicerota.markdown]: Good point, changed.

File: source/_integrations/fireservicerota.markdown (hunk 10 above)
Comment (cyberjunky) [reply source/_integrations/fireservicerota.markdown]: Fixed

File: source/_integrations/fireservicerota.markdown (hunk 18 above)
Comment (cyberjunky) [reply source/_integrations/fireservicerota.markdown]: Nice, fixed.

**Comment by cyberjunky** [comment]
> Thanks for the PR @cyberjunky, I've left some review comments. Could you please take a look? Thanks +1

@frenck I think I have addressed all suggestions.

File: source/_integrations/fireservicerota.markdown [hunk 19]
```diff
@@ -0,0 +1,226 @@
...
+| `latitude` | The Latitude of the incident.|
+| `longitude` | The Longitude of the incident.|
+| `address_type` | Type of address, e.g. `home`.|
+| `formatted_address` | Address in string format.|
+
+### Duty Binary Sensor
+
+| Value | Description |
+| ----- | ----------- |
+| `value` | `on` = on duty. `off` = no duty scheduled.|
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: As mentioned before, I really think we do not need a table here.
It makes more sense to have a written introduction about the sensor and mention the values in that description. (Removing the whole table).

File: source/_integrations/fireservicerota.markdown [hunk 20]
```diff
@@ -0,0 +1,226 @@
...
+| `assigned_function_ids` | Function id's, e.g. `540`.|
+| `skill_ids` | Skill id's, e.g. `6, 8`.|
+| `type` | Type, e.g. `standby_duty`.|
+| `assigned function` | Assigned function, e.g. `Chauffeur`.|
+
+### Incident Response Switch
+
+| Value | Description |
+| ----- | ----------- |
+| `value` | `on` = response is acknowledged. `off` = the response is rejected.|
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: As mentioned before, I really think we do not need a table here.
It makes more sense to have a written introduction about the sensor and mention the values in that description. (Removing the whole table).

File: source/_integrations/fireservicerota.markdown [hunk 21]
```diff
@@ -0,0 +1,226 @@
...
+      attribute: message_to_speech_url
+    condition: not
+    conditions:
+      condition: state
+      entity_id: sensor.incidents
+      attribute: message_to_speech_url
+      state: "None"
+    action:
+      - service: media_player.play_media
+        data_template:
```
Comment (frenck) [inline source/_integrations/fireservicerota.markdown]: You might want to test this, it seems invalid now.

File: source/_integrations/fireservicerota.markdown (hunk 20 above)
Comment (cyberjunky) [reply source/_integrations/fireservicerota.markdown]: @frenck I change them, what do you think?

**Comment by frenck** [comment]
@cyberjunky I kinda meant just the first tables with the single value 😓 (the state, not the attributes)

**Comment by cyberjunky** [comment]
> @cyberjunky I kinda meant just the first tables with the single value sweat (the state, not the attributes)

Haha. LOL oke, wil revert and adapt the value only.

**Review by frenck — APPROVED** [review]
Thanks, @cyberjunky! 👍

**Pull request merged**